src/actions/guides/database/querying-deleting.cr

```diff
@@ -360,15 +360,31 @@ class Guides::Database::QueryingDeleting < GuideAction
     `SELECT AVG(users.age) FROM users`
 
     ```crystal
+    # This will return a Float64 | Nil.
+    # The value will be nil if there are no records.
     UserQuery.new.age.select_average
+
+    # This will return a Float64.
+    # The value will be 0 if there are no records.
+    UserQuery.new.age.select_average!
     ```
 
     `SELECT SUM(users.age) FROM users`
 
     ```crystal
+    # Returns Nil if there are no records
```

Comment:
Maybe this could be split into two lines and with a bit more detail:

```
# Returns an Int64 for integer columns, or a Float64 for float columns
# Returns 'nil' if there are no records
```